syntaxes/omg.idl.tmLanguage.json

```diff
@@ -572,11 +573,109 @@
             "name": "annotation.omg.idl",
             "patterns": [{
                 "name": "storage.type.annotation.omg.idl",
-                "match": "@\\w+(\\(.*\\)){0,1}"
+                "match": "@\\w+\\s*(\\(.*\\)){0,1}"
             }]
+        },
+        "interface_def": {
+            "name": "meta.storage.interface.omg.idl",
+            "begin": "(interface)\\s+(\\w+)\\s*\\{",
```

Comment:
Interfaces can inherit from other interfaces; I think you have that syntax in structs.

Reply:
Done.